components/automate-ui/src/app/entities/infra-nodes/infra-nodes.effects.ts

```diff
@@ -156,6 +156,7 @@ export class InfraNodeEffects {
   updateNodeEnvironmentSuccess$ = createEffect(() =>
     this.actions$.pipe(
       ofType(NodeActionTypes.UPDATE_ENVIRONMENT_SUCCESS),
+       // eslint-disable-next-line no-empty-pattern    
       map(({ }: UpdateNodeEnvironmentSuccess) => new CreateNotification({
```

Comment:
```suggestion
      map(() => new CreateNotification({
```
since we are not using the param we can remove it entirely instead of suppressing the error